test/skip_list_test.js

```diff
@@ -192,11 +192,11 @@ describe('SkipList', () => {
 
     it('should behave like a JS array', () => {
       jsc.assert(jsc.forall(jsc.bless({generator: makeSkipListOps}), function (ops) {
-        let levels = ops.filter(op => op.hasOwnProperty('insertAfter')).map(op => op.level)
+        let levels = ops.filter(op => op.hasOwnProperty('insertAfter')).map(op => op.level) // eslint-disable-line
```

Comment:
using hasOwnProperty on an object directly is problematic because if someone passes an object like this: `{ hasOwnProperty: "HAXXXOR" }` it will cause a crash. Instead the recommendation is to defensively call op.prototype.hasOwnProperty()